go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/yvasiyarov/go-metrics v0.0.0-20140926110328-57bccd1ccd43 // indirect
 	github.com/yvasiyarov/gorelic v0.0.0-20141212073537-a9bba5b9ab50 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20140908184405-b21fdbd4370f // indirect
-	golang.org/x/net v0.4.0 // indirect
+	golang.org/x/net v0.3.0 // indirect
```

Comment:
@lucacome This dependency is downgraded. Do you know the reason?

Reply:
I've noticed too, but I couldn't figure out why

Reply:
@lucacome Can you run `go mod why golang.org/x/net` to check? BTW, I've merged all your other PRs.

Reply:
@shizhMSFT I've rebased after all the other merges and now `v0.4.0` is back...